LocalSettings.php

```diff
@@ -2457,6 +2457,9 @@
 			[ 'newtablinks', 'wikiwalk' ],
 			'_self' => [ 'sametablinks' ]
 		],
+		'sdiywiki' => '_blank' => [ 'extiw' ],
```

Comment:
Add a `[` sign in front of `'_blank'` and a `]` sign after `'extiw' ]` (but before the comma!). Repeat this for the other lines as well.